### PR TITLE
Upgrade intx to 0.2

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: lib/evmone/execution.cpp [hunk 1]
```diff
@@ -212,20 +212,19 @@ void op_iszero(execution_state& state, instr_argument) noexcept
 
 void op_and(execution_state& state, instr_argument) noexcept
 {
-    // TODO: Add operator&= to intx.
-    state.item(1) = state.item(0) & state.item(1);
+    state.item(1) &= state.item(0);
```
Comment (axic) [inline lib/evmone/execution.cpp]: Tiny speedups incoming? :)

Comment (chfast) [reply lib/evmone/execution.cpp]: No. The intx API cleanup.

Comment (axic) [reply lib/evmone/execution.cpp]: Yeah but potentially intx could do the `and` in place without copies -> speed up if sufficiently large number of ands/shifts/etc. are done.

Comment (chfast) [reply lib/evmone/execution.cpp]: Maybe there is some space for improvement, but `x &= y` is defined `x = x & y` do no functional change here. Syntax sugar only.

Comment (axic) [reply lib/evmone/execution.cpp]: But it would be possible to improve it in `intx` without the need to change it here, after this was merged.

Comment (chfast) [reply lib/evmone/execution.cpp]: Yes, that's correct.

**Pull request merged**